packages/edit-site/src/components/editor/index.js

```diff
@@ -138,6 +149,11 @@ export default function Editor( { isLoading, onClick } ) {
 		};
 	}, [] );
 
+	const { setSectionRootBlockName } = useDispatch( blocksStore );
+	useEffect( () => {
+		setSectionRootBlockName( getSectionRootBlockName( context?.postType ) );
+	}, [ context, setSectionRootBlockName ] );
```

Comment:
Can we trigger this wherever the `postType` gets set instead of in a `useEffect` in the component?

Reply:
Yes we actually have to as this effect is reaaaaaly slowing down everything.

Reply:
Removed the effect.